src/eval-frame/global-state-extras.js

```diff
@@ -0,0 +1 @@
+export const IODIDE_EVALUATION_RESULTS = {};
```

Comment:
IODIDE_EVALUATION_RESULTS is basically a global variable that is used to manage a part of state that we cannot put in the redux store. i thought we should put that (and anything else like it) into it's own file rather than having a stateful thing defined in some random place